test/crdsvalidation/dataplane_validatingadmissionpolicy_test.go

```diff
@@ -1,32 +1,104 @@
 package crdsvalidation
 
 import (
+	"path"
 	"testing"
 	"time"
 
+	"github.com/go-logr/zapr"
 	"github.com/samber/lo"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	ctrl "sigs.k8s.io/controller-runtime"
 
+	commonv1alpha1 "github.com/kong/kubernetes-configuration/v2/api/common/v1alpha1"
 	operatorv1beta1 "github.com/kong/kubernetes-configuration/v2/api/gateway-operator/v1beta1"
+	konnectv1alpha1 "github.com/kong/kubernetes-configuration/v2/api/konnect/v1alpha1"
 	"github.com/kong/kubernetes-configuration/v2/test/crdsvalidation/common"
 
 	"github.com/kong/kong-operator/modules/manager/scheme"
+	"github.com/kong/kong-operator/pkg/utils/test"
 	"github.com/kong/kong-operator/test/envtest"
-	"github.com/kong/kong-operator/test/helpers/kustomize"
+	"github.com/kong/kong-operator/test/helpers/helm"
 )
 
 const (
-	// KustomizePathValidatingPolicies is the path to the Kustomize directory containing the validation policies.
-	KustomizePathValidatingPolicies = "config/default/validating_policies/"
+	// ChartPath is the path relative to the project to the kong-operator chart
+	ChartPath = "charts/kong-operator"
+
+	// ValidationPolicy_ contain validation policies path relative to the kong-operator chart
+	ValidationPolicyDataplane = "templates/validation-policy-dataplane.yaml"
```

Comment:
This seems unused. I've checked out the repo on this branch and can't see any references to this

```suggestion
```